src/drive/web/modules/viewer/Panel/useBlocks.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+
+import { hasCertifications } from 'drive/web/modules/viewer/helpers'
+import Certifications from './Certifications'
+
+// TODO add connector block
+const blocksSpecs = file => ({
+  certifications: {
+    condition: hasCertifications(file),
+    component: <Certifications file={file} />
+  }
+})
+
+const useBlocks = ({ file }) => {
+  const blocks = []
+
+  Object.values(blocksSpecs(file)).map(block => {
+    block.condition && blocks.push(block.component)
+  })
+
+  return blocks
+}
```

Comment:
This does not seem to be a hook (it is not using any other hook), can it be renamed into `getPanelBlocks` ? Otherwise, the developer might think it's a hook, and that it can only be used in React context ?

Reply:
Ok merci pour la précision 👍